src/components/charts/list/DiscoverCharts.tsx

```diff
@@ -105,18 +108,26 @@ function DiscoverChartList() {
     const [isGrid, setIsGrid] = useState<boolean>(false)
     const [showGitOpsWarningModal, toggleGitOpsWarningModal] = useState(false)
     const [clickedOnAdvance, setClickedOnAdvance] = useState(null)
+
+
     const noChartAvailable: boolean = chartList.length > 0 || searchApplied || selectedChartRepo.length > 0
     isLeavingPageNotAllowed.current = !state.charts.reduce((acc: boolean, chart: ChartGroupEntry) => {
         return (acc = acc && chart.originalValuesYaml === chart.valuesYaml)
     }, true)
 
+    useEffect(()=>{
```

Comment:
Can we merge this into the other useEffect instead of creating new one for the same